cpp/CMakeLists.txt

```diff
@@ -349,16 +349,16 @@ add_library(cugraph SHARED
     src/community/ECG.cu
     src/cores/core_number.cu
     src/traversal/two_hop_neighbors.cu
-    src/snmg/blas/spmv.cu
-    src/snmg/link_analysis/pagerank.cu
+    #src/snmg/blas/spmv.cu
+    #src/snmg/link_analysis/pagerank.cu
     src/utilities/cusparse_helper.cu
     src/utilities/graph_utils.cu
-    src/snmg/utils.cu
+    #src/snmg/utils.cu
     src/components/connectivity.cu
     src/centrality/katz_centrality.cu
     src/centrality/betweenness_centrality.cu
-    src/snmg/degree/degree.cu
-    src/snmg/COO2CSR/COO2CSR.cu
+    #src/snmg/degree/degree.cu
+    #src/snmg/COO2CSR/COO2CSR.cu
     src/nvgraph/arnoldi.cu
```

Comment:
Can we just delete these instead of commenting out? We can see github history when we need to get these back.

Reply:
They should be removed since we are not going bad to SNMG.  OPG is the future :-)

Reply:
(already did that)